### PR TITLE
docs: update contribution docs

### PR DESCRIPTION
**Pull request by yuluo-yx**
- For #376

### CODE REVIEW

**Comment by yuluo-yx** [comment]
ptal @Agility6 @LinuxSuRen

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -42,7 +44,7 @@ Other usage of this:
 * `/debug/pprof/heap?seconds=10`
 * `/debug/pprof/goroutine/?debug=0`
 
-## Skywalking
+## SkyWalking
```
Comment (LinuxSuRen) [inline CONTRIBUTION.md:47]: Good cache.

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -69,6 +72,18 @@ docker run -p 12800:12800 -p 9412:9412 \
     docker.io/apache/skywalking-oap-server
 ```
 
+## First contribution
+
+## First contribution
```
Comment (LinuxSuRen) [inline CONTRIBUTION.md:77]: Duplicated with line 75

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Only a minor issue needs to be fixed.

File: CONTRIBUTION.md (hunk 2 above)
Comment (yuluo-yx) [reply CONTRIBUTION.md]: done!

**Review by LinuxSuRen — APPROVED** [review]

**Pull request merged**